### PR TITLE
When SOAP Body is returned blank <soap:body/>, there were failures in…

### PR DESCRIPTION
**Pull request by vidishdatta**
When SOAP Body is returned blank <soap:body/>, there were failures in processing it.

### CODE REVIEW

**Comment by vidishdatta** [comment]
Hi @jsdevel , @herom - Requesting your review. Please let me know if I can provide any further info.

Also, how do we usually publish this back to NPM and at what intervals? The current version on NPM is 16.0

Thanks!

File: test/client-test.js [hunk 1]
```diff
@@ -718,6 +718,35 @@ var fs = require('fs'),
...
+            server = null;
+            assert.ok(!err);
+            assert.ok(!responseSoapHeaders);
+            assert.ok(result);
+            assert.ok(body);
+            done();
+          });
+        }, baseUrl);
+      });
+
```
Comment (jsdevel) [inline test/client-test.js]: please remove this empty line.

**Comment by jsdevel** [comment]
Looks good @vidishdatta.  One minor comment then I think we're good to go.

**Comment by jsdevel** [comment]
Thanks @vidishdatta !

**Comment by vidishdatta** [comment]
Hi @jsdevel , Thanks for accepting changes. This is my first contribution to open source.

Just wanted to understand - When do we usually release it to NPM registry and when do we usually create a release for this repository?

Thanks Again!

**Comment by jsdevel** [comment]
We usually get to releases every 2 weeks or so.

**Pull request merged**